modules/virt-virtctl-commands.adoc

```diff
@@ -29,70 +29,201 @@ To view a list of global command options that you can use with any `virtctl` com
+
+.`virtctl vmexport` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
+|`virtctl vmexport create <vmexport_name> --vm\|snapshot\|pvc=<object_name>`
+|Create a `VirtualMachineExport` custom resource (CR) to export a volume from a VM, VM snapshot, or PVC.
+
+* `--vm`: Exports the PVCs of a VM.
+* `--snapshot`: Exports the PVCs contained in a `VirtualMachineSnapshot` CR.
+* `--pvc`: Exports a PVC.
+* Optional: `--ttl=1h` specifies the time to live. The default duration is 2 hours.
+
+|`virtctl vmexport delete <vmexport_name>`
+|Delete a `VirtualMachineExport` CR manually.
+
+|`virtctl vmexport download <vmexport_name> --output=<output_file> --volume=<volume_name>`
+|Download the volumes defined in a `VirtualMachineExport` CR.
+
+* `--output` specifies the file format. Example: `disk.img.gz`.
+* `--volume` specifies the volume to download. This flag is optional if only one volume is available.
+
+Optional:
+
+* `--keep-vme` retains the `VirtualMachineExport` CR after download. The default behavior is to delete the `VirtualMachineExport` CR after download.
+* `--insecure` enables an insecure HTTP connection.
+
+|`virtctl vmexport download <vmexport_name> --<vm\|snapshot\|pvc>=<object_name> --output=<output_file>`
+|Create a `VirtualMachineExport` CR and then download the volumes defined in the CR.
+|===
+
+[id='vm-memory-dump_{context}']
+== VM memory dump
+
+You can use the `virtctl memory-dump` command to output a virtual machine (VM) memory dump on a PVC. You can specify an existing PVC or use the `--create-claim` flag to create a new PVC.
+
+.Prerequisites
+
+* The PVC volume mode must be `FileSystem`.
+* The PVC must be large enough to contain the memory dump.
++
+The formula for calculating the PVC size is `(VMMemorySize + 100Mi) * FileSystemOverhead`, where `100Mi` is the memory dump overhead.
+
+* You must enable the hot plug feature gate in the `HyperConverged` custom resource by running the following command:
++
+[source,terminal]
+----
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv \
+  --type json -p '[{"op": "add", "path": "/spec/featureGates", \
+  "value": "HotplugVolumes"}]'
+----
+
+.Downloading the memory dump
+
+Download the memory dump by using the `virtctl vmexport download` command:
+
+[source,terminal]
+----
+$ virtctl vmexport download <vmexport_name> \
+  --vm\|pvc=<object_name> --output=<output_file>
+----
+
+.`virtctl memory-dump` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
+|`virtctl memory-dump get <vm_name> --claim-name=<pvc_name>`
+|Save the memory dump of a VM on a PVC. The memory dump status is displayed in the `status` section of the `VirtualMachine` resource.
+
+Optional:
+
+* `--create-claim` creates a new PVC with the appropriate size. This flag has the following options:
+
+** `--storage-class=<storage_class>`
+** `--access-mode=ReadWriteOnce`
```

Comment:
currently can be either Readwriteonce or ReadWriteMany (just not readonly)

Reply:
Understood. Will fix.